app/views/ReportOnlineMarketplaceAlternativeChargeView.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("reportOnlineMarketplaceAlternativeCharge.title"))) {
+
+    @formHelper(action = routes.ReportOnlineMarketplaceAlternativeChargeController.onSubmit(mode), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukRadios(
+            RadiosViewModel.yesNo(
+                field = form("value"),
+                legend = LegendViewModel(messages("reportOnlineMarketplaceAlternativeCharge.heading")).asPageHeading()
```

Comment:
@chandraAndem The default page size that .asPageHeading() brings is "govuk-label--xl" the one that we need is "govuk-label--l", DST FE uses "govuk-label--l" as page heading style. 
```suggestion
                legend = LegendViewModel(messages("reportOnlineMarketplaceAlternativeCharge.heading")).asPageHeading(LabelSize.Large)
```

Reply:
Done